force-app/main/default/lwc/edaSettingsNavigationSection/edaSettingsNavigationSection.html

```diff
@@ -0,0 +1,16 @@
+<template>
+    <div class="slds-nav-vertical__section">
```

Comment:
Calling out that normally we would use a vertical navigation component. Instead, we are appending to the design pattern used for slds vertical navigation alongside our UI innovation on settings, which will need to treat individual menu items as one comprehensive whole. We may reverse course before the end of the release, but I wanted to call out why we are not using that base component.